benchs/tpch/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -eu
+set -o pipefail
+
+TAR_VER=$(tarantool -v | grep -e "Tarantool" |  grep -oP '\s\K\S*')
+numaopts="numactl --membind=1 --cpunodebind=1 --physcpubind=6,7,8,9,10,11"
+base_dir=$(pwd)
+
+cd /opt/tpch
+make TPC-H.db
+
+killall tarantool tpcc_load 2>/dev/null || true
```

Comment:
`tpcc_load`? it's not very relevant to this script. Was it intentional, or just copy-paste?

Reply:
it was copy-paste. removing.